engine/src/flutter/shell/platform/windows/text_input_plugin.cc

```diff
@@ -328,12 +337,11 @@ void TextInputPlugin::HandleMethodCall(
           TextRange(composing_base, composing_extent), cursor_offset);
     }
   } else if (method.compare(kSetMarkedTextRect) == 0) {
-    // TODO(loicsharma): Remove implicit view assumption.
-    // https://github.com/flutter/flutter/issues/142845
-    FlutterWindowsView* view = engine_->view(kImplicitViewId);
+    FlutterWindowsView* view = engine_->view(view_id_);
     if (view == nullptr) {
       result->Error(kInternalConsistencyError,
-                    "Text input is not available in Windows headless mode");
+                    "Text input is not available when corresponding view "
+                    "cannot be found");
```

Comment:
Nit: consider adding the view ID to these error messages for debugging purposes. I don't feel strongly